mcs/class/corlib/System.Security.Cryptography/CryptoConfig.Mobile.cs

```diff
@@ -197,10 +196,10 @@ public static object CreateFromName (string name, params object[] args)
 				name = "System.Security.Cryptography.X509Certificates.X509Chain, System";
 				break;
 			case "aes":
-#if FULL_AOT_DESKTOP  // TODO: why is this special cased? we could use AesManaged like other full AOT profiles
-				name = "System.Security.Cryptography.AesCryptoServiceProvider, System.Core";
-#else
+#if MONOTOUCH || XAMMAC
```

Comment:
this would need to add XAMMAC_4_5 to keep the previous meaning. But I think we should just go with AesManaged on all !FEATURE_CRYPTO_CONFIGURABLE platforms.

Reply:
The file was not used before with XAMMAC_4_5 and it returned null for the value (that's another bug).

.NET returns `AesCryptoServiceProvider` and I don't think we need to break the compatibility here. My preference would be to remove `if MONOTOUCH || XAMMAC` instead but that needs more work

Reply:
Oh right, I got confused by all the conditions here :)